src/jvmMain/kotlin/software/momento/kotlin/sdk/internal/DataGrpcStubsManager.kt

```diff
@@ -0,0 +1,51 @@
+package software.momento.kotlin.sdk.internal
+
+import grpc.cache_client.ScsGrpcKt
+import io.grpc.ClientInterceptor
+import io.grpc.ManagedChannel
+import io.grpc.ManagedChannelBuilder
+import software.momento.kotlin.sdk.auth.CredentialProvider
+import java.io.Closeable
+import java.util.concurrent.TimeUnit
+import kotlin.time.Duration.Companion.minutes
+
+internal class DataGrpcStubsManager(credentialProvider: CredentialProvider) : Closeable {
+    private val channel: ManagedChannel
```

Comment:
When we factor out the common code I think we'll be able to make an abstract version of the stubs manager in common. The grpc classes used here are all in the api dependency.